### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by teikmeout**
This PR will do the following:

- Remove policy and users docs, solves INVENIO-152
- Update docs homepage, solves INVENIO-153
- Add invenio-cli cheatsheet
- Add project orientation content and updated links
- Update Ruby Version to 3.0.5, solves INVENIO-151
- Update gem github-pages
- Update homepage Links
- Update docker documents
- Update Look and Feel documentation
- solves INVENIO-150

To test this PR make sure to:
- install newer version of Ruby with `rbenv install`
- install bundler with `gem install bundler`
- reinstall gems with `bundler install --redownload`
- run the project locally with `bundle exec jekyll serve`

To test this PR please check Github Actions [`pages-build-deployment`](https://github.com/nyudlts/ultraviolet/actions/workflows/pages/pages-build-deployment) to ensure it can run on this pipeline. I'm assuming that the GH action will read the `.ruby-version` file, but have no direct way to prove that it will build accordingly.

Thank you for our review!
Please let me know if there are changes needed :)

### CODE REVIEW

**Comment by teikmeout** [comment]
We might want to pause this one since I'm seeing https://pages.github.com/versions/#:~:text=ruby,2.7.4
and I think it might be better to stick to what github pages is using...

**Comment by teikmeout** [comment]
Context and Motivation:
- when trying to locally setup and run this documentation on M1 mac I ran into many installation issues.
- I found that we are using an outdated Ruby version 2.7.4 (newest 2.7.7) with endOfLife date on [March 2023](https://www.ruby-lang.org/en/downloads/branches/)
- I tried using Ruby 2.7.7 and installation was successful. This was a good sign. I would prefer to update out of this version into something with future support.
- I tried using Ruby 3.0.5 and after adding `webrick` (gem was removed in Ruby > 3) `bundle install` was successful. Another good sign. This is when I thought this was ready to become a PR.
- I was worried about our CI/CD pipelines not following the newer version of Ruby since we use a community Github Action. So I kept on researching.

What I have learned:
- Github pages is a Ruby + Jekyll Project that uses [Ruby 2.7.4 and Jekyll 3.9.3](https://pages.github.com/versions/)
- Github pages does not specify what version of Ruby to install either in their [Test your Local Site documentation](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll#prerequisites) or their [Create a Site documentation](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll)
- Github pages recommends using `github/pages-gem` that is a gem that helps setup a development environment quickly but also maps and [locks a ruby version and a jekyll version](https://github.com/github/pages-gem#whats-versioned). The mention this is valuable since different Ruby versions have different default tools, recommending to actually stick to older versions of Ruby and Jekyll.
- Also weirdly enough: When starting a GithubPages/Jekyll site from scratch, they instruct you to install [(any) Ruby version](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll) followed by installing any version of Jekyll, only to later comment out the jekyll gem from the gemfile and [replace it with the `github-pages` gem](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll#:~:text=Add%20the%20github,VERSION%22%2C%20group%3A%20%3Ajekyll_plugins). This could lead to starting your site with a new Jekyll version that might be higher than the `github-pages` gem contains.
- Github pages in its documentation makes an inconspicuous note when running into [`webrick` issues](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll#:~:text=Note%3A%20If,jekyll%20serve.) implying it's accounted for using Ruby > 3.0?? Yet when reading deeper into their docs you can see how theirs an intention of keeping consistency in using Ruby 2.7.4 and Jekyll 3.9.3
- Funny enough [Jekyll 4.3.0](https://jekyllrb.com/news/2022/10/20/jekyll-4-3-0-released/) already includes the `webrick` gem in the Gemfile making the migration to newer Ruby > 3.0 even easier.

What happens next?
- option 1: we keep this PR, update to newer Ruby for our local development setup and we deal with the inconsistencies at deployment.
- option 2: we stick to what Github pages uses for consistency and change the Ruby version used in this project.
- option 3: we make a PR to the Github pages repo? and try to push for newer usage of Ruby > 3 and Jekyll > 4?

Thank you for your insight and please let me know your thoughts and if you'd like me to revert the Ruby update to stick to what Github pages uses.

**Comment by VickyRampin** [comment]
I don't think anyone is tied down to using Jekyll either, so option #4 might be to abandon our current format and move the content that is relevant to [Sphinx](https://www.sphinx-doc.org/en/master/) or some other more accepted docs format/tooling.

**Comment by debverhoff** [comment]
Jonathan, Thanks for digging in and coming up with these options.

I don't have any valuable input about updating Ruby locally or otherwise,
and defer to Kate et al. As per Vicky's note that we could also move to
another docs tool, I think that is a fair option too. I rather like having
the docs near to the code, but maybe I am overthinking that convenience
when a link could suit the purpose.

What do others think? It will be good to keep momentum on this since we
have some documentation in the wings.

**Comment by teikmeout** [comment]
I agree with what @VickyRampin says. 
A Jekyll website for documentation seemed a little over-engineered.
I'm used to seeing a `README.md`, and a `CONTRIBUTING.md` that use [Github-flavored-Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax), that are version controlled with git, that live near the project and can be viewed by everyone since the project is public and all Markdown files render stylized on Github. 
Having to maintain also a Jekyll site adds the burden of knowing Ruby and Jinja Templates. Dockerizing can be a solution to reduce installation complexity, and that could be the second step on this PR, but still requires things like building and deploying as part of updating documentation. I would rather not need to maintain Ruby gems and just update a Markdown file. 
I did all this digging to allow the team to make an informed decision. I'm down for whatever direction we want to move.
I also don't want to undo someone else's work while not knowing the full intention of having this documentation to be deployed in github pages. Maybe you guys have some better context. 

LMK what direction we steer this ship 👍

**Comment by teikmeout** [comment]
Ok, as we discussed before, I removed Jekyll and Github Pages.
This developer documentation is now only Markdown.
I recommend we also turn off and remove the Github Action for deploying to Github Pages as well as remove the configuration tying our URL to the site.
The rest should be self explanatory: I hope I organized the documentation into something a little more sensical and useful. Please feel free to comment on this PR any other changes you want made to the documentation.

Thank you for your time reviewing this ticket.

File: docs/SETUP.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+- Fully Dockerized Mode: Containerized application and services (good for a quick preview).  
+- Partially Dockerized Mode: Local dev tools runnig parts of InvenioRDM with containerized services (good for developers) also considered [instance development](https://inveniordm.docs.cern.ch/develop/getting-started/instance-development/#integrating-react-development-modules).
+
+> From the InvenioRDM docs: In the future we aim to move to a **fully containerized development environment.**  
+> More on the [InvenioRDM Architecture](https://inveniordm.docs.cern.ch/develop/architecture/)  
+
+## System Requirements
+[back to top](#setup)
+
+> As of Jan 2023, Ultraviolet uses InvenioRDM version 7 that is behind a couple of versions from the current InvenioRDM v11 released in Jan 2023.
```
Comment (ekate) [inline docs/SETUP.md:26]: it uses 3.8.12

File: README.md [hunk 2]
```diff
@@ -1,35 +1,23 @@
...
+# Ultraviolet 
+
 [![ci:test](https://github.com/nyudlts/ultraviolet/workflows/ci:test/badge.svg)](https://github.com/nyudlts/ultraviolet/actions)
 [![ci:test:docs](https://github.com/nyudlts/ultraviolet/workflows/ci:test:docs/badge.svg)](https://nyudlts.github.io/ultraviolet/)
 
-UltraViolet is NYU Libraries' platform for long-term access to a broad range of NYU-related digital content, including open access data associated with research and scholarly output and data licensed for use by members of the NYU community. This repository is the codebase for UltraViolet, a local deployment of the [InvenioRDM framework](https://inveniordm.docs.cern.ch/).
-
-## Getting Started
+UltraViolet is NYU Libraries' platform for long-term access to a broad range of NYU-related digital content, including open access data associated with research and scholarly output and data licensed for use by members of the NYU community.  
+UltraViolet is a local deployment of the [InvenioRDM framework](https://inveniordm.docs.cern.ch/) maintained by CERN.
```
Comment (ekate) [inline README.md:7]: In my understanding InvenioRDM is not a framework, it uses Invenio framework to build turn-key repository

File: docs/develop/look-feel.md [hunk 3]
```diff
@@ -47,29 +87,22 @@ on YYYY-MM-DD
 from v6.0.4
```
Comment (ekate) [inline docs/develop/look-feel.md:87]: We can delete this

File: docs/CONTRIBUTE.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+## How we Develop
+
+<!-- - [UltraViolet 💜 codebase](https://github.com/nyudlts/ultraviolet) -->
+- Learn about Architecture, System Requirements, Setup steps in our [DEVELOP documentation](./DEVELOP.md)
+
+
+### Branching Strategy
+
+- `main` represents truth
+- branch naming convention: `<ticketNumber>/<summary-title>`
```
Comment (lhenze) [inline docs/CONTRIBUTE.md:50]: I thought we separated these with a hyphen?

File: docs/SETUP.md (hunk 1 above)
Comment (teikmeout) [reply docs/SETUP.md:26]: Corrected the version 👍

File: README.md (hunk 2 above)
Comment (teikmeout) [reply README.md:7]: Ok, clarified language for this one too

File: README.md [hunk 5]
```diff
@@ -1,35 +1,23 @@
...
+NYU Libraries contributes to InvenioRDM as part of their open-source community.
 
+## Getting Started as a User of Ultraviolet
 
-We recommend you visit our [Documentation Wiki](https://nyudlts.github.io/ultraviolet/) to get started. 
-You can also check [InvenioRDM documentation](https://inveniordm.docs.cern.ch/)
-Have Questions? Please submit them at the [Issues tab](https://github.com/nyudlts/ultraviolet/issues).
+For information about NYU's UltraViolet repository service, including policies and how to submit data, see: [https://guides.nyu.edu/ultraviolet](https://guides.nyu.edu/ultraviolet).  
 
+## Getting Started as a Developer on Ultraviolet
```
[the remark this answers is not in the document]
Comment (teikmeout) [reply README.md:14]: Ok, added a link for that

File: docs/CHEATSHEETS.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    - `invenio-cli run` - start local dev server, exit with [CTRL+C]
+        - `invenio-cli run --services` enable dockerized services
+        - `invenio-cli run --no-services` disable dockerized services
+        - `invenio-cli run --debug
+- **Individual level application**
+    - `invenio-cli packages install $HOME/projects/invenio/`
+        - runs installation for pip packages of said project
+    - `invenio-cli assets build` - injects variables and transpiles
+    - `invenio-cli assets watch` - watch mode, exit with [CTRL+C]
+        - can only watch existing files, if creating a new file stop your server and `invenio-cli assets build` before watching again.
```
[the remark this answers is not in the document]
Comment (teikmeout) [reply docs/CHEATSHEETS.md:37]: Technically it doesn't. It's mostly a preference between starting/stoping the command to do more stuff in that terminal if you need it. I'll add a comment so people can choose their path.

File: docs/CONTRIBUTE.md (hunk 4 above)
Comment (teikmeout) [reply docs/CONTRIBUTE.md:50]: You are completely right! correcting

**Comment by teikmeout** [comment]
Closing down this PR since it has already been added into the `development` branch of this project.

**Pull request closed without merging**